SparkyFitnessMobile/src/hooks/useAppBootstrap.ts

```
@@ -0,0 +1,53 @@
+        if (cancelled) return;
+
+        const route: BootstrapRoute = config ? 'Tabs' : 'Onboarding';
+        setInitialRoute(route);
+        setLinkingEnabled(route === 'Tabs');
+      } catch (error) {
+        if (cancelled) return;
+        const message = error instanceof Error ? error.message : String(error);
+        addLog(`[App] Failed to load active server config on startup: ${message}`, 'ERROR');
+        setInitialRoute('Onboarding');
+      } finally {
+        if (cancelled) return;
```

Biome (2.5.6)

[error] Unsafe usage of 'return'.

(lint/correctness/noUnsafeFinally)